src/data/supervisor/supervisor.ts

```diff
@@ -18,3 +34,24 @@ export interface Supervisor {
   resolution: HassioResolution;
   os: HassioHassOSInfo;
 }
+
+const subscribeSupervisorEventUpdates = (conn: Connection, store) =>
+  conn.subscribeEvents(
+    (event) => store.setState(event, true),
```

Comment:
the store should contain the supervisor, not the event data?

Reply:
should it?

Reply:
so, you dont want to use overwrite here